test/gtest/vec_add.cpp

```diff
@@ -147,9 +151,9 @@ struct VecAddTest : public ::testing::TestWithParam<VecAddTestCase>
 
         std::string params = options.GenerateFor(miopen::kbp::HIP{});
 
-        uint totalElements  = vec_size;
-        int threadsPerBlock = threads_per_block;
-        int blocksPerGrid   = (totalElements + threadsPerBlock - 1) / threadsPerBlock;
+        size_t totalElements = vec_size;
+        int threadsPerBlock  = threads_per_block;
```

Comment:
Do not convert `size_t` to `int`. That's a `narrowing` conversion. Either keep everything as `int or unsigned int` or keep everything as `size_t`.